jsonschema_lexer/data/keywords/draft2020-12.json

```diff
@@ -0,0 +1,64 @@
+{
+  "dialect": "https://json-schema.org/draft/2020-12/schema",
+  "identifier": "$id",
+  "keywords": [
+    "$schema",
+    "$id",
+    "$ref",
+    "$defs",
+    "$comment",
+    "$dynamicAnchor",
+    "$dynamicRef",
+    "$anchor",
+    "$vocabulary",
+    "oneOf",
+    "allOf",
+    "anyOf",
+    "if",
+    "then",
+    "else",
+    "not",
+    "properties",
+    "patternProperties",
+    "additionalProperties",
+    "dependentSchemas",
+    "propertyNames",
+    "prefixItems",
+    "contains",
+    "items",
+    "title",
+    "description",
+    "default",
+    "deprecated",
+    "examples",
+    "readOnly",
+    "writeOnly",
+    "type",
+    "enum",
+    "const",
+    "minLength",
+    "maxLength",
+    "pattern",
+    "maximum",
+    "exclusiveMinimum",
+    "multipleOf",
+    "exclusiveMaximum",
+    "minimum",
+    "dependentRequired",
+    "minProperties",
+    "maxProperties",
+    "required",
+    "minItems",
+    "maxItems",
+    "minContains",
+    "maxContains",
+    "uniqueItems",
+    "format",
+    "unevaluatedItems",
+    "unevaluatedProperties",
+    "contentEncoding",
+    "contentMediaType",
+    "contentSchema",
+    "format_assertion"
```

Comment:
What's this?

Reply:
Ah my bad, looks like while compiling the list of keywords somehow noted the heading as well. Sorry :(

<img width="663" alt="Screenshot 2024-03-02 at 9 21 22 PM" src="https://github.com/python-jsonschema/jsonschema-lexer/assets/75855708/038480c6-729e-4459-a593-d438d85b6ce6">